reconstruction/cvt/src/main/java/org/jlab/rec/cvt/trajectory/Helix.java

```diff
@@ -53,7 +53,7 @@ public Helix(double dca, double phi_at_doca, double curvature, double Z0, double
     }
 
     public Helix(double pt, double d0, double phi0, double Z0, double tandip, int q, double xb, double yb, double solenoidMag) {
-        double curvature = -(double)q*Constants.LIGHTVEL*solenoidMag/pt;
+        double curvature = (double)q*Constants.LIGHTVEL*solenoidMag/pt * Math.signum(Constants.getSolenoidScale());

```

Comment:
Since both the light velocity and the solenoid scale from the Constant class are used, maybe could also get the solenoid field magnitude from there instead if having that as an argument